src/main/scala/com.mediative.eigenflow/EigenflowBootstrap.scala

```diff
@@ -60,9 +61,16 @@ trait EigenflowBootstrap {
   private def stopSystem(system: ActorSystem)(code: Int): Unit = {
     Try(messagingSystem.stop())
 
-    system.terminate().onComplete {
-      case Success(_) => System.exit(code)
-      case Failure(_) => System.exit(1)
+    // in case if termination process hangs
+    try {
```

Comment:
We still want to halt the system no matter what exception happened.
But printing the stacktrace would be useful, will add it.